cli_reference/cli_manager/cli-manager-release-notes.adoc

```diff
@@ -0,0 +1,29 @@
+:_content-type: ASSEMBLY
+[id="cli-manager-release-notes_{context}"]
+= {cli-manager} release notes
+include::_attributes/common-attributes.adoc[]
+:context: cli-manager-release-notes
+
+toc::[]
+
+The {cli-manager} allows you to install CLI plugins in both connected and disconnected environments.
+
+These release notes track the development of the {cli-manager} for {product-title}.
+
+//For an overview of the {run-once-operator}, see xref   :../../../nodes/pods/run_once_duration_override/index.adoc#run-once-about_run-once-duration-override-about[About the {cli-manager}].
+
```

Comment:
You'll need to add the technology-preview.adoc snippet file to all of these assemblies.